src/test/scala/com/fulcrumgenomics/umi/CallDuplexConsensusReadsTest.scala

```diff
@@ -58,14 +58,20 @@ class CallDuplexConsensusReadsTest extends UnitSpec {
     checkClpAnnotations[CallDuplexConsensusReads]
   }
 
-  it should "run fail if AB-R1s are not on the same strand ads BA-R2s" in {
+  it should "not generate a consensus if AB-R1s are not on the same strand ads BA-R2s" in {
     val builder = new SamBuilder(readLength=10)
     builder.addPair(name="ab1", start1=100, start2=200, attrs=Map(MI -> "1/A"), bases1="AAAAAAAAAA", bases2="AAAAAAAAAA")
```

Comment:
It would be nice to be explicit about the strands here even though it's mirroring the defaults.